backend/server/api.go

```diff
@@ -152,6 +153,16 @@ func bindAPIRoutes(r *mux.Router, db *database.Connection, contentStore contents
 		return services.ListTagsForOperation(r.Context(), db, i)
 	}))
 
+	route(r, "GET", "/api/operations/{operation_slug}/evidence", jsonHandler(func(r *http.Request) (interface{}, error) {
+		dr := dissectJSONRequest(r)
+		tlq, _ := helpers.ParseTimelineQuery(dr.FromQuery("q").AsString())
+		i := services.ListEvidenceForOperationInput{
+			OperationSlug: dr.FromURL("operation_slug").Required().AsString(),
+			Filters:       tlq,
+		}
+		return services.ListEvidenceForOperation(r.Context(), db, i)
```

Comment:
The dissectJSONRequest function is really just a wrapper around accessing parts of the request. If those parts of the parsing don't work, then it'll _record_ an error. You still need to check if that error was produced. 

_Technically_, I think we can be pretty confident that `dr.Error` will be nil, as the URL parameter should be present, and the query parameter is optional. However, I'd still like to see the check here because by not including it we're setting ourselves up for an error down the road.

Reply:
```suggestion
		if dr.Error != nil {
			return nil, dr.Error
		}
		return services.ListEvidenceForOperation(r.Context(), db, i)
```